cmake/modules/hunter_check_toolchain_for_define.cmake

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2016 NeroBurner
+# All rights reserved.
+
+include(hunter_test_string_not_empty)
+
+function(hunter_check_toolchain_for_define definition output_var)
+  # the toolchain path must be set
+  hunter_test_string_not_empty("${HUNTER_TOOLCHAIN_ID_PATH}")
+  set(TOOLCHAIN_INFO_FILE "${HUNTER_TOOLCHAIN_ID_PATH}/toolchain.info")
+
+  # read toolchain.info file into memory
+  file(READ "${TOOLCHAIN_INFO_FILE}" toolchain_contents)
+
+  # Convert file contents into a CMake list (where each element in the list
+  # is one line of the file)
+  string(REGEX REPLACE ";" "\\\\;" toolchain_contents "${toolchain_contents}")
```

Comment:
You can use `file(STRINGS)`:
* https://cmake.org/cmake/help/v3.0/command/file.html